project/core/models.py

```diff
@@ -161,3 +161,16 @@ def __str__(self):
     class Meta:
         unique_together = ('profile', 'date')
         ordering = ['date']
+
+class HappyWord(models.Model):
+    happy_content = models.CharField(max_length=200, null=False)
```

Comment:
그냥 content라고 해도 괜찮을 것 같아요 
```suggestion
    content = models.CharField(max_length=200, null=False)
```